packages/editor-ui/src/components/Telemetry.vue

```diff
@@ -39,7 +39,7 @@ export default defineComponent({
 			this.init();
 		},
 		currentUserId(userId) {
-			if (this.isTelemetryEnabled) {
+			if (this.isTelemetryEnabled && this.$telemetry) {
```

Comment:
Why would `this.$telemetry` be undefined?

Reply:
stupid e2e was breaking. Was a shot in the dark based on a vague error. It did not work 🤷🏽